sortinghat/core/api.py

```diff
@@ -336,6 +336,10 @@ def move_identity(ctx, from_id, to_uuid):
 
     identity = find_identity(from_id)
 
+    if identity.id == identity.uidentity.uuid:
+        msg = "'from_id' cannot match its UniqueIndentity. Try with 'merge'"
```

Comment:
Something like `'from_id' is a unique identity and it cannot be moved; use 'merge' instead` might work better. What do you think?

Reply:
The documentation of `move_identity` and `mv` command should be updated too.

Reply:
Done!